src/presenter/board-presenter.js

```diff
@@ -29,27 +29,39 @@ export default class BoardPresenter {
 
   #renderEvent(event, allDestinations, offersByType) {
     allDestinations = this.destinations;
-    offersByType = this.#eventsModel.getOffersByType(this.#boardEvents[0].type);
+    offersByType = this.#eventsModel.getOffersByType(event.type);
```

Comment:
Здесь если хочешь вычислить офферы по типу 1 раз то 
```suggestion
    const offersByType = this.#eventsModel.getOffersByType(event.type);
```
и дальше используешь ниже по коду, передавать в функцию renderEvent не надо